clouddq/classes/dq_rule.py

```diff
@@ -69,15 +73,18 @@ def to_dict(self: DqRule) -> dict:
 
         """
 
-        return dict(
-            {
-                f"{self.rule_id}": {
-                    "rule_type": self.rule_type.name,
-                    "params": self.params,
-                    "rule_sql_expr": self.resolve_sql_expr(),
-                }
+        d = {
+            f"{self.rule_id}": {
+                "rule_type": self.rule_type.name,
+                "params": self.params,
+                "rule_sql_expr": self.resolve_sql_expr(),
             }
-        )
+        }
+
+        if self.dimension:
+            d[self.rule_id]["dimension"] = self.dimension
```

Comment:
True, one code path less. The reason I did it like this is that it affects the final result: If you don't have the key in the config, the summary will have NULL in the dimension column. Now, it will have "None". But I removed it.